openapi/components/schemas/Customer.yaml

```
@@ -103,6 +103,10 @@ properties:
       or an accepted credit file document.
     type: boolean
     readOnly: true
+  organizationId:
+    readOnly: true
+    allOf:
+      - $ref: ./OrganizationId.yaml
   _links:
     type: array
     description: The links related to resource.
```

nitpick: Where possible, we should avoid starting descriptions with "the"

This can be tiresome to read

```suggestion
    description: Related resource links.
```